README.md

```diff
@@ -1,5 +1,11 @@
 # Cost report generator
 
+## Requirements
+
+- Install [jq](https://stedolan.github.io/jq/download/)
+- Install [parallel](https://github.com/flesler/parallel)
```

Comment:
You link to a reimplementation of _GNU Parallel_. You should link to the original implementation at https://www.gnu.org/software/parallel/ instead, which is the one I used and will be found on any distribution’s repositories ( and in Homebrew’s ).